stdlib/2and3/fractions.pyi

```diff
@@ -0,0 +1,89 @@
+# Stubs for fractions
+# See https://docs.python.org/3/library/fractions.html
+
+from typing import Optional, TypeVar, Union, overload
+from numbers import Complex, Rational
+from decimal import Decimal
+import sys
+
+_TComplex = TypeVar('_TComplex', bound=Complex)
+_TOtherNum = TypeVar('_TOtherNum', int, float, Decimal)
+_TNum = TypeVar('_TNum', int, float, Decimal, Complex)
+
+@overload
+def gcd(a: _TComplex, b: _TComplex) -> _TComplex: ...
```

Comment:
I don't know what the docs say, but the module doesn't seem to support complex arguments here.